airbyte-integrations/connectors/source-sentry/source_sentry/sentry.yaml

```diff
@@ -0,0 +1,107 @@
+definitions:
+  page_size: 50
+  schema_loader:
+    type: JsonSchema
+    file_path: "./source_sentry/schemas/{{ options.name }}.json"
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_pointer: []
+  requester:
+    type: HttpRequester
+    name: "{{ options['name'] }}"
+    url_base: "https://{{ config.hostname }}/api/0/"
+    http_method: "GET"
+    authenticator:
+      type: "BearerAuthenticator"
+      api_token: "{{ config.auth_token }}"
+  paginator:
+    type: LimitPaginator
+    url_base: "*ref(definitions.requester.url_base)"
+    page_size: "*ref(definitions.page_size)"
+    limit_option:
+      inject_into: "request_parameter"
+      field_name: ""
```

Comment:
what does it mean to have an empty field_name? is this correct?

Reply:
the right solution here is probably for `limit_option` and `page_token_option` to be optional. [left a comment](https://github.com/airbytehq/airbyte/issues/15575#issuecomment-1212543875)